src/modules/privateChat/components/doctors-list-card.component.tsx

```diff
@@ -21,36 +21,36 @@ interface DoctorsListCardProps {
  * @param param0
  * @returns
  */
-const DoctorsListCardItem = () => {
+const DoctorsListCardItem = (doctor: any) => {
```

Comment:
@oluwadamilarey

The doctor in the DoctorsListCardItem params list is wrong, the props passed to a component is an object, it should be:

`const DoctorsListCardItem = ({doctor}: {doctor:  User}) =>`

That way inside the component you wont need to do anything like {doctor.doctor}, that is clearly wrong, if anything it should be {props.doctor}, but since you should be destructuring the doctor from the props object, you won't need the props.